doc/version-control-fields.rst

```diff
@@ -1,95 +0,0 @@
-Version Control System Fields
-=============================
-
-.. _vcs-fields:
-
-Most of the version control system (VCS) fields types are common to both
-``source-repository`` and ``source-repository-package`` stanzas.
-
-.. list-table::
-    :header-rows: 1
-    :widths: 30 30 40
-
-    * - Field Name
-      - source-repository (head|this)
-      - source-repository-package
-    * - type
-      - [x]
-      - [x]
-    * - location
-      - [x]
-      - [x]
-    * - branch
-      - [x]
-      - [x]
-    * - tag
-      - [x]
-      - [x]
-    * - subdir
-      - [x] (0 or 1)
-      - [x] (0 or 1 for each dependency)
-    * - module (CVS only)
-      - [x]
-      - [_]
-    * - post-checkout-command
-      - [_]
-      - [x]
```

Comment:
It would be a shame to loose this table, 4 of the fields are the same but 3 are not. It is good to be able to see that.